pkg/api/types.go

```diff
@@ -584,6 +584,8 @@ type DownwardAPIVolumeFile struct {
 	Path string `json:"path"`
 	// Required: Selects a field of the pod: only annotations, labels, name and  namespace are supported.
 	FieldRef ObjectFieldSelector `json:"fieldRef"`
+	// Required: Selects a field of the pod; only name and namespace are supported.
+	ConfigDataRef *ConfigDataSelector `json:"configDataRef"`
```

Comment:
We don't.  @tmrts, I thought we agreed this was going to be a distinct volume plugin similar to secrets?